core/templates/dev/head/editor/EditorServices.js

```diff
@@ -1522,7 +1522,7 @@ oppia.factory('stateEditorTutorialFirstTimeService', [
         }
 
         if (_currentlyInFirstVisit) {
-          $rootScope.$broadcast('openEditorTutorial');
+          $rootScope.$broadcast('openEditorTutorialFirstTime');
```

Comment:
Let's change the name of this event to "enterEditorForTheFirstTime". It's more correct that way, since the tutorial might not actually get opened.